Sheheroes_WatchOS/pubspec.lock

```diff
@@ -63,7 +63,7 @@ packages:
       name: async
```

Comment:
Please remove this lock file from commit, but keep this in your local.
See Why: https://dart.dev/guides/libraries/private-files#pubspeclock